File: app/src/main/java/com/tosh/poolandroid/View/AppIntro/ThirdFragment.java
package com.tosh.poolandroid.View.AppIntro;


import android.content.Intent;
import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;


import com.tosh.poolandroid.R;
import com.tosh.poolandroid.View.LoginActivity;


public class ThirdFragment extends Fragment {
    private Button getStartedBtn;

    public ThirdFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.fragment_third, container, false);

        Button btnStarted = view.findViewById(R.id.btn_getStarted);

        btnStarted.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getActivity(), LoginActivity.class);
                startActivity(intent);
                getActivity().finish();
            }
        });


        return view;
    }

}

File: app/src/main/java/com/tosh/poolandroid/view/appIntro/ThirdFragment.kt
package com.tosh.poolandroid.view.appIntro

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.TextView
import androidx.fragment.app.Fragment
import com.tosh.poolandroid.R
import com.tosh.poolandroid.view.activity.LoginActivity

class ThirdFragment : Fragment() {
    private val getStartedBtn: Button? = null
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? { // Inflate the layout for this fragment
        val view = inflater.inflate(R.layout.fragment_third, container, false)
        val btnStarted: TextView = view.findViewById(R.id.btn_getStarted)
        btnStarted.setOnClickListener {
            val intent = Intent(activity, LoginActivity::class.java)
            startActivity(intent)
            activity!!.finish()
        }
        return view
    }
}